packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -11,7 +11,6 @@
 import android.app.KeyguardManager;
 import android.content.Context;
 import android.content.Intent;
-import android.content.pm.PackageManager;
 import android.hardware.fingerprint.FingerprintManager;
```

Comment:
If we're no longer reporting fingerprint specifically, do we even still need this? Doesn't `androidx.biometric.BiometricPrompt` handle fallback to fingerprint for us (which we're currently doing manually) if we just remove our API level gating on the call to `AuthenticationHelper`?

Reply:
This should probably be a follow-up PR, I just wanted to flag it to make sure I'm understanding the relationship between the new query mechanism and the manual fallback we're currently using.

Reply:
It is currently only used by the implementation of `authenticate`, so for this PR i'd indeed consider it out of scope, but from a quick look it does seem like something that should be removable.